test/livebook_teams/web/session_live_test.exs

```diff
@@ -22,24 +23,91 @@ defmodule LivebookWeb.Integration.SessionLiveTest do
       id = hub.id
       personal_id = Livebook.Hubs.Personal.id()
 
-      Session.subscribe(session.id)
       {:ok, view, _} = live(conn, ~p"/sessions/#{session.id}")
 
-      assert Session.get_data(session.pid).notebook.hub_id == personal_id
+      assert Session.get_notebook(session.pid).hub_id == personal_id
 
       view
       |> element(~s/#select-hub-#{id}/)
       |> render_click()
 
       assert_receive {:operation, {:set_notebook_hub, _, ^id}}
-      assert Session.get_data(session.pid).notebook.hub_id == hub.id
+      assert Session.get_notebook(session.pid).hub_id == hub.id
+    end
+
+    test "changes the notebook hub when the org deletes the user",
+         %{conn: conn, user: user, node: node, session: session} do
+      Livebook.Hubs.Broadcasts.subscribe([:connection, :secrets])
+      Livebook.Teams.Broadcasts.subscribe([:clients])
+
+      Application.put_env(:livebook, :hub_disconnect_backoff, 500)
```

Comment:
I think you can set this in config/test.exs and apply for the whole suite?

Reply:
We doesn't need it anymore